semgrep/semgrep/commands/ci.py

```diff
@@ -132,14 +132,22 @@ def fix_head_if_github_action(metadata: GitMeta) -> Iterator[None]:
 @scan_options
 @click.option(
     "--audit-on",
-    envvar=["SEMGREP_AUDIT_ON"],
+    envvar="SEMGREP_AUDIT_ON",
     multiple=True,
     type=str,
     hidden=True,
 )
+@click.option(
+    "--app-url",
+    default="https://semgrep.dev",
+    envvar="SEMGREP_APP_URL",
```

Comment:
Can we have semgrep-agent handle the translation between this ENV var and SEMGREP_URL env var that semgrep expects?